package.json

```diff
@@ -19,7 +19,8 @@
   "dependencies": {
     "custom-card-helpers": "^1.6.4",
     "home-assistant-js-websocket": "^5.6.0",
-    "lit-element": "^2.3.1"
+    "lit-element": "^2.3.1",
+    "jsonpath-plus": "^5.0.7"
   },

```

Comment:
Why is this needed?

Reply:
You're right, this is not needed, I first tried to allow jsonpath to specify keys so that you was able to reach nested attributes. But it makes things overcomplicated so finally not used it.